### PR TITLE
Merging arduino use-case version1

### PR DESCRIPTION
**Pull request by parduino**
Version 1 of arduino use-case material

### CODE REVIEW

File: docs/arduino/usecase.md [hunk 1]
```diff
@@ -1,89 +1,248 @@
...
+Site response analysis for liquefiable soils is fundamental in the estimation of demands on civil infrastructure including buildings and lifelines. Current state of the art in numerical methods in geotechnical engineering require the use of advance constitutive models and fully couple nonlinear finite element (FEM) tools. Advanced constitutive models require calibration of material parameters based on experimental tests. These parameters include uncertainties that in turn propagate to uncertenties in the estimation of demands. The products included in this use-case provide simple examples showing how to achieve site response analysis including parameter identification and uncertainty quantification using SimCenter tools and the DesignSafe cyber infrastructure.
+
+[//]: <![Propagation of vertical waves in site response analysis](img/SRschematic2.PNG){width=50%}>
```
Comment (kks32) [inline docs/arduino/usecase.md]:
```suggestion
```

File: docs/arduino/usecase.md [hunk 2]
```diff
@@ -1,89 +1,279 @@
...
 ## Background 
 
 ### Citation and Licensing
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite [Arduino, P. et al. (2022)](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
 
 * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).
```
Comment (kks32) [inline docs/arduino/usecase.md]:
```suggestion
* This software is distributed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.html).  
```

File: docs/arduino/usecase.md [hunk 3]
```diff
@@ -1,89 +1,279 @@
...
+Site response analysis for liquefiable soils is fundamental in the estimation of demands on civil infrastructure including buildings and lifelines. Current state of the art in numerical methods in geotechnical engineering require the use of advance constitutive models and fully couple nonlinear finite element (FEM) tools. Advanced constitutive models require calibration of material parameters based on experimental tests. These parameters include uncertainties that in turn propagate to uncertenties in the estimation of demands. The products included in this use-case provide simple examples showing how to achieve site response analysis including parameter identification and uncertainty quantification using SimCenter tools and the DesignSafe cyber infrastructure.
+
+<p align="center">
+<img src="img/SRschematic2.PNG" alt="Propagation of vertical waves in site response analysis" width="600"/>
```
Comment (kks32) [inline docs/arduino/usecase.md]:
```suggestion
<img src="../img/SRschematic2.PNG" alt="Propagation of vertical waves in site response analysis" width="600"/>
```

File: docs/arduino/usecase.md [hunk 4]
```diff
@@ -1,89 +1,279 @@
...
 ### Citation and Licensing
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite [Arduino, P. et al. (2022)](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
 
 * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
 
 ### Description
```
Comment (kks32) [inline docs/arduino/usecase.md]:
```suggestion
## Description 
```

Comment (kks32) [reply docs/arduino/usecase.md]: Please use second level H2 so they appear on the sidebar

File: docs/arduino/usecase.md (hunk 3 above)
Comment (kks32) [reply docs/arduino/usecase.md]: Image url is one level up

File: docs/arduino/usecase.md [hunk 5]
```diff
@@ -1,89 +1,279 @@
-# TITLE for Use Case Documentation Page
+# From constitutive parameter calibration to site response analysis
 
 **Authors, Affiliations**
```
Comment (kks32) [inline docs/arduino/usecase.md]:
```suggestion
```

**Review by kks32 — CHANGES_REQUESTED** [review]
Could you please update the doc with the suggested changes. Thanks
!

**Comment by parduino** [comment]
Made suggested modifications

**Comment by parduino** [comment]
Hi Krishna,

  Silly question on use case document.
Why do you suggest to use


Propagation of vertical waves in site response analysis


If the img/ folder is in the same location as the usecase.md file… ??
Or are you planning to have a different location for the img folders??
I would use

Propagation of vertical waves in site response analysis

I may be missing something here.. ☺

Pedro

**Comment by parduino** [comment]
Hi Krishna,

  I made some additional changes based on your comments. It really help me see your page to undestand what you want.
One of the problems seems to be related to how headings will show in the master page… (now I see this problem).
For this purpose I changed my usecase.md page in such a way 4 headings show at the H2 level:

a)       Background

b)      Site response notebook (completed for this version)

c)       Parameter Calibration Notebook (in preparation)

d)      Propagation of uncertainty notebook (in preparation)

For the images the problem of relative is respect to what. I believe your ../img is because you are starting from the main page. Is this correct?
Can make any change you consider appropriate.

Pedro

From: Pedro Arduino
Sent: Saturday, April 16, 2022 12:21 PM
To: DesignSafe-CI/DS_Use_Case_template ***@***.***>; Krishna Kumar ***@***.***>
Subject: RE: [DesignSafe-CI/DS_Use_Case_template] Merging arduino use-case version1 (PR #18)

Hi Krishna,

  Silly question on use case document.
Why do you suggest to use


Propagation of vertical waves in site response analysis


If the img/ folder is in the same location as the usecase.md file… ??
Or are you planning to have a different location for the img folders??
I would use

Propagation of vertical waves in site response analysis

I may be missing something here.. ☺

Pedro

**Comment by parduino** [comment]
Hi Pedro,

Sorry for my slow reply, had lots of classes yesterday.

I'm working through the changes. Thanks for changing the headings, they 
look good now.

For the images, because you use direct html, the webpage for your 
usecase is 
https://deploy-preview-18--ds-use-case-preview.netlify.app/arduino/usecase/ 
and the image is actually located in `/arduino/img` so from the current 
`/arduino/usecase/` we have to go to the parent directory `/ardunio/` 
using `../img` to access the image folder.

I have edit access to your repo, so I can make these changes. Just 
explaining here for why we need to make this change. You don't have to 
do anything at the moment.

Thank you!!

- Krishna

**Review by kks32 — APPROVED** [review]

**Pull request merged**